src/components/ShowDateTimeSidebar/index.jsx

```diff
@@ -81,8 +81,8 @@ export const ShowDateTimeSidebar = ({
                     title={title} 
                     description={description} 
                     location={singlePerformance.platform} 
-                    start={singlePerformance.dateTimeStart} 
-                    end={singlePerformance.dateTimeEnd ? singlePerformance.dateTimeEnd : processedStartDateTime.setHours(processedStartDateTime.getHours() + 2)} 
+                    start={new Date(singlePerformance.dateTimeStart)} 
+                    end={new Date(singlePerformance.dateTimeEnd ? singlePerformance.dateTimeEnd : processedStartDateTime.setHours(processedStartDateTime.getHours() + 2))} 
```

Comment:
Thanks for the review request. It allows me to explore parts of the codebase I hadn't seen yet, in bite-sized chunks.

Looks good to me but if I was to comment on anything, it would be to replace the end date ternery with an "or" statement as we're essentially repeating ourselves while checking for a truthy value, so...
```
new Date(singlePerformance.dateTimeEnd || processedStartDateTime.setHours(processedStartDateTime.getHours() + 2))
```
...would have the same outcome without the repetition.